src/tests/components/forecastsummeries.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
```

Comment:
Theres a typo in the filename, in should be ForecastSummaries (as its testing a component it should be written in PascalCase)